lib/pbio/sys/program_load.c

```diff
@@ -82,6 +82,15 @@ pbio_error_t pbsys_program_load_get_user_data(uint32_t offset, uint8_t **data, u
     return PBIO_SUCCESS;
 }
 
+void pbsys_program_load_set_bluetooth_disabled(bool bluetooth_disabled) {
+    map->header.bluetooth_disabled = bluetooth_disabled;
```

Comment:
This doesn't seem to be working and I'm having trouble understanding why. Any ideas would be appreciated.

If I replace this code with reading/writing to a file-static variable then everything behaves as expected (though of course the setting is not persisent). As it is, it always reads false, both during startup and after I attempt to modify it.